packages/design-system/src/types/keyboard.ts

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Shouldn't this file exist inside the keyboard folder? Are the types required outside of the design system package? And even if they are, I still think they should be defined inside the keyboard folder actually.

Reply:
So far we've added a `types.ts` file in the `src` folder. Since one `types.ts` would get too large then I've created a `types` folder instead. If there's a different preference, should we do the same in all packages perhaps?
To me it makes sense to have consistency.

Is there a best practice for this? Maybe let's create a follow-up issue then where we'd move all the non-exported types out of the `types.ts` files?